requirements-dev.txt

```diff
@@ -1,9 +1,9 @@
 # Internationalization and Localization requirements
```

Comment:
@kaizoku, the version of Django in `test_requirements.txt` hasn't been updated to the latest 2.2.x version. Can you check and update that as well?

Reply:
I can't get `pip-compile` to run successfully after updating this requirement in `test_requirements.in` as it includes base and test requirements from the xblock-sdk, which doesn't support Django 2.2 yet. Should we wait for https://github.com/edx/xblock-sdk to update?

Reply:
@kaizoku, do you know which requirements in the `xblock-sdk` repo are causing this issue? If we are using the `0.1.7` release and that has this issue, can you check if this has been fixed in the `master` branch after the release? If yes, we can request edX to tag a new release so that we can use it. Otherwise, we should get in touch with edX about this and report this issue as the https://github.com/edx/xblock-sdk/pull/165 PR was for adding Django 2.2 support.

Reply:
@lgp171188 I was testing with master before, but their `requirements/base.txt` which problem-builder inherits [specifies `django==1.11.26`](https://github.com/edx/xblock-sdk/pull/165/files#diff-7fe11226cff646f5d9f35faa76217059R16) in the PR, and has [the same requirement in the current master](https://github.com/edx/xblock-sdk/blob/74bf5d6010b37e591a687e1e227263d689412279/requirements/base.txt#L17).

`requirements/base.in` specifies `Django>=1.11`, but pip-compile sets that to `==1.11.26` for some reason.

Reply:
I managed to get this working, but I have to [upgrade `xblock-sdk/requirements/base.in` and the requirement for `fs-s3fs` to `0.1.8`](https://github.com/edx/xblock-sdk/blob/74bf5d6010b37e591a687e1e227263d689412279/requirements/base.in#L9) in order to fix the issues with resolving `six` dependencies.

Should I create a PR for xblock-sdk to update that as well?

Reply:
@kaizoku, yes, please go ahead and create the `xblock-sdk` PR. If we can find the right person  in edX to ping for this, we can get it reviewed and merged soon. @bradenmacdonald can you tell us who from edX requested us to update problem-builder to support Django 2.2?